doc/man3/SSL_get_certificate.pod

```diff
@@ -0,0 +1,51 @@
+=pod
+
+=head1 NAME
+
+SSL_get_certificate, SSL_get_privatekey - retrieve TLS/SSL certificate and
+private key
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ X509 *SSL_get_certificate(const SSL *s);
+ EVP_PKEY *SSL_get_privatekey(const SSL *s);
+
+=head1 DESCRIPTION
+
+SSL_get_certificate() returns a pointer to the B<X509> object representing the
+certificate which represents the local peer's identity. For servers, it returns
+the server's certificate. If a server has multiple certificates (for different
+algorithms, for example RSA and ECDSA), the certificate in actual use is
+returned. For clients, it returns the client certificate being used, if any.
```

Comment:
I don't think this isn't quite correct. Before the handshake it returns the last certificate that was added to the SSL/SSL_CTX or NULL if none were added yet. During the handshake it selects a certificate to use for the connection and from that point on it returns the selected certificate.

Reply:
I still don't think this is quite right. I would merge the text you added under RETURN VALUES into this text. The text about "multiple certificates" at the moment is quite misleading in the case where it is called before the handshake. Perhaps adding something like "or the most recently added certificate if called before a certificate has been selected". Or something like that.